src/types/flags.ts

```diff
@@ -7,6 +7,7 @@ export type Flags = {
   systemProfileHiddenFields: boolean;
   helpFooter: boolean;
   cedar508Requests: boolean;
+  notifyMultipleRecipients: boolean;
```

Comment:
This takes care of the remaining work for [EASI-1983](https://jiraent.cms.gov/browse/EASI-1983), incidentally.